### PR TITLE
Key storage guide and python example

### PR DESCRIPTION
**Pull request by henriquecd**
This pull request resolves dojot/ma-client-libs#1

### CODE REVIEW

File: Key_Storage.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Secure Key storage
+
+The Application Key is received upon the application registration and grants access to the mutual authentication and crypto channel functions. After received, the key must be protected since it will be used to identify the application as legitimate. In order to do that, it is highly recomended that the application maintainer stores the key in an adequate way.
```
Comment (rascaraficci) [inline Key_Storage.md]: 'recomended' to 'recommended'

File: Key_Storage.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+This guide presents solutions and good practices to obtain a secure key storage.
+
+## OWASP recomendations
+
+The [OWASP](https://www.owasp.org) is an organization focused in improving software security and have a [series of recomendations](https://www.owasp.org/index.php/Cryptographic_Storage_Cheat_Sheet) regarding cryptographic storage. In the case of key storage, some recomendations are:
```
Comment (rascaraficci) [inline Key_Storage.md]: 'focused in' to 'focused on'
'and have a' to 'and has a'

File: Key_Storage.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Secure Key storage
+
+The Application Key is received upon the application registration and grants access to the mutual authentication and crypto channel functions. After received, the key must be protected since it will be used to identify the application as legitimate. In order to do that, it is highly recommended that the application maintainer stores the key in an adequate way.
```
Comment (mmagr) [inline Key_Storage.md]: ... received upon ~~the~~ application registration ...
... grants access to ~~the~~ mutual authentication ...

File: Key_Storage.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+This guide presents solutions and good practices to obtain a secure key storage.
+
+## OWASP recomendations
+
+The [OWASP](https://www.owasp.org) is an organization focused on improving software security and has a [series of recomendations](https://www.owasp.org/index.php/Cryptographic_Storage_Cheat_Sheet) regarding cryptographic storage. In the case of key storage, some recomendations are:
```
Comment (mmagr) [inline Key_Storage.md]: ~~The~~ OWASP is ...

**Pull request merged**